tests/plugins/panel/panel.js

```diff
@@ -0,0 +1,28 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: panel */
+
+( function() {
+	'use sctrict';
+
+	bender.editor = {};
+
+	bender.test( {
+		// #2035
+		'test panel src is empty': function() {
+			if ( !CKEDITOR.env.edge ) {
+				assert.ignore();
+			}
+
+			var editor = this.editor,
+				panel = new CKEDITOR.ui.panel( CKEDITOR.document, {
+					forceIFrame: true
+				} ),
+				html = panel.render( editor ),
+				src = html.match( /src="[^"]*"/i )[ 0 ];
+
+			src = src.substring( 5, src.length - 1 );
```

Comment:
You could simplify it simply by using regexp group for `[^"]*` part in https://github.com/ckeditor/ckeditor-dev/blob/294ada7/tests/plugins/panel/panel.js#L21